### DISCUSSION TITLE
Text size on some apps is very large, unsure why

### DISCUSSION THREAD

**Discussion by rykugur** [i3/i3#5778] [Q&A]
hey folks. I upgraded my GPU a few months back and ever since the text size on some apps, and i3 titlebars, has been massive. I don't think it's related to the GPU upgrade, but hey I've been surprised in the past.

Here's a few screenshots showing what I mean.

![image](https://github.com/i3/i3/assets/2609898/6ca11b76-47d3-42a5-a8a6-5930eeff299f)
![image](https://github.com/i3/i3/assets/2609898/64f26726-b059-43a2-94be-ba68eed45fc2)

GPU is a 4070 (nvidia-dkms), x11. This doesn't happen on all apps (i.e. unsure if it's Qt, gtk apps only).

Any ideas why and what I can do to fix it?

**Comment by okraits** [comment]
I could imagine that the scaling has changed with the graphics driver. i3 uses pango for text rendering. I would try to change the scaling. You might have a look at https://wiki.archlinux.org/title/HiDPI.
